test/parallel/test-tls-basic-validations.js

```diff
@@ -103,8 +103,16 @@ assert.throws(
   const inputBuffer = Buffer.from(arrayBufferViewStr.repeat(8), 'utf8');
   for (const expectView of common.getArrayBufferViews(inputBuffer)) {
     const out = {};
+    let expected;
+    if (expectView.constructor === Uint8Array) {
+      expected = Buffer.from(expectView);
+    } else {
+      expected = Buffer.from(expectView.buffer.slice(),
+                             expectView.byteOffset,
+                             expectView.byteLength);
+    }
```

Comment:
```suggestion
    const expected = Buffer.from(expectView.buffer,
                                 expectView.byteOffset,
                                 expectView.byteLength);
```

An `Uint8Array` is still an `ArrayBufferView`.